### DISCUSSION TITLE
How to ensure that Sioyek re-renders after switching displays?

### DISCUSSION THREAD

**Discussion by nickel8448** [ahrm/sioyek#596] [Q&A] [answered]
Hello,

I use an external display to read PDFs using Sioyek and whenever I disconnect the display to read on my laptop display the highlights get out of place but are still linked to the intended text. I have to restart Sioyek to make it re-render the PDF.

Does anyone know how can I get Sioyek to re-render as soon as I change displays?

I hope I am using the terminology right to explain this. If it helps, I use Sioyek on an M1 MacBook.

Btw, love the tool. Thanks for creating it.

**Comment by ahrm** [comment, marked as answer]
The `reload` command doesn't work?

**Reply by nickel8448** [reply]
It doesn't for some reason. I tried it again.

Seeing this even after using `reload`.
<img width="1269" alt="Screenshot 2023-01-14 at 21 16 53" src="https://user-images.githubusercontent.com/4462306/212497365-bace9eab-d2b2-4a78-a02d-0a6480c42893.png">

**Reply by nickel8448** [reply]
Please let me know if I can provide any kind of logs to help with this.

**Reply by nickel8448** [reply]
Hey @ahrm, please let me know if I can help with some logs or some other way to help in reproducing this issue.

**Reply by ahrm** [reply]
I don't think logs are going to help a lot here. Maybe share your exact screen configuration:
* what is the resolution of each screen
* what is the display scaling of each screen

**Reply by nickel8448** [reply]
Apologies for not providing an update.
For some reason I am not facing this issue anymore.

* I was using it on a 4K monitor and then moving to my laptop screen 3024 x 1964 resolution on the laptop
* As far as I understand the question - I was on 'More Space' scaling on Mac for both monitors.

**Comment by nickel8448** [comment]
I am unable to close this question. Can someone else please let me know how to close it
